components/metal/CapabilitiesSection.vue

```diff
@@ -49,35 +51,32 @@
 </template>
 
 <!-- TODO: Refactor using Composition API once Mixins are refactored -->
-<script lang="ts">
-import { Component } from "vue-property-decorator";
+<script setup lang="ts">
 import { MetalCapability, METAL_CAPABILITIES } from "~/constants/metalContent";
 import { useScrollBetweenSections } from "~/composables/useScrollBetweenSections";
 
 // TODO: Review how this work
 const { activeSection } = useScrollBetweenSections();
-
-@Component
-export default class CapabilitiesSection {
-  capabilities = METAL_CAPABILITIES;
-
-  isActiveImage(item: MetalCapability, index: number): boolean {
-    return (
-      item.title === this.activeSection ||
-      (this.activeSection === "" && index === 0)
-    );
-  }
-
-  isVideo(url: string): boolean {
-    const extension = url.substring(url.length - 4);
-    return extension === ".mp4";
-  }
-}
+const capabilities = METAL_CAPABILITIES;
+
+const isActiveImage = (item: MetalCapability, index: number): boolean => {
+  return (
+    item.title === activeSection.value ||
+    (activeSection.value === "" && index === 0)
+  );
+};
+
+const isVideo = (url: string): boolean => {
+  const extension = url.substring(url.length - 4);
+  return extension === ".mp4";
+};
 </script>
 
 <style lang="scss" scoped>
 @use "~/assets/scss/carbon.scss";
-@import "~carbon-components/scss/globals/scss/typography";
+@use "~/assets/scss/helpers/index.scss" as qiskit;
+
+// @import "~carbon-components/scss/globals/scss/typography";
```

Comment:
I believe we can remove this.

Reply:
Resolved https://github.com/Qiskit/qiskit.org/pull/3019/commits/b02185b3cea9147da743470ae4c3688d0657e26c